Library/Homebrew/test/utils/svn_spec.rb

```diff
@@ -0,0 +1,46 @@
+require "utils/svn"
+
+describe Utils do
+  describe "#self.svn_available?" do
+    before(:each) do
+      if described_class.instance_variable_defined?(:@svn)
+        described_class.send(:remove_instance_variable, :@svn)
+      end
+    end
+
+    it "returns svn version if svn available" do
+      expect(described_class.svn_available?).to be_truthy
+    end
+  end
+
+  describe "#self.svn_remote_exists" do
+    it "returns true when svn is not available" do
+      allow(Utils).to receive(:svn_available?).and_return(false)
+      expect(described_class.svn_remote_exists("blah")).to be_truthy
+    end
+
+    context "when svn is available" do
+      before do
+        allow(Utils).to receive(:svn_available?).and_return(true)
+        url.mkpath
+      end
+
+      let(:url) { mktmpdir/"blah" }
+
+      it "returns false when remote does not exist" do
+        expect(described_class.svn_remote_exists(url)).to be_falsey
+      end
+
+      it "returns true when remote exists", :needs_network do
+        remote = "http://amethyst.yui.ne.jp/svn/pc98/np2/trunk/"
```

Comment:
Using the above url, I'm getting the following: 
```
Error validating server certificate for 'https://github.com:443':
 - The certificate is not issued by a trusted authority. Use the
   fingerprint to validate the certificate manually!
Certificate information:
 - Hostname: github.com
 - Valid: from Mar 10 00:00:00 2016 GMT until May 17 12:00:00 2018 GMT
 - Issuer: DigiCert SHA2 Extended Validation Server CA, www.digicert.com, DigiCert Inc, US
 - Fingerprint: D7:9F:07:61:10:B3:92:93:E3:49:AC:89:84:5B:03:80:C1:9E:2F:8B
(R)eject, accept (t)emporarily or accept (p)ermanently?
```
In the tests, how do I ensure that I don't need to manually answer the above posed question of accepting the host?